assets/js/app/Data.js

```diff
@@ -4,8 +4,60 @@
 
 "use strict";
 
+/* A named collection of data sources used to draw a plot*/
+LocusZoom.DataSources = function() {
```

Comment:
Would it make more sense to nest this inside the Data object, as in `LocusZoom.Data.Sources`? This would keep a more consistent *file* architecture where each app file completely defines one and only one class in the singleton's namespace. Otherwise this is the only file (so far) that defines two: `LocusZoom.Data` and `LocusZoom.DataSources`.